core/src/main/java/io/questdb/griffin/engine/ops/CreateMatViewOperationImpl.java

```diff
@@ -414,7 +429,8 @@ public void validateAndUpdateMetadataFromModel(
                 }
                 model.setIsDedupKey();
                 // Copy column names into builder to be validated later.
-                copyBaseTableColumnNames(column.getAst(), queryModel, baseTableName, baseKeyColumnNames);
+                copyBaseTableColumnNames(column.getAst(), queryModel, baseTableName,
```

Comment:
nit: staircase formatting

Reply:
fixed in https://github.com/questdb/questdb/pull/5491/commits/6065f7b26ea6fb7d33cd48b37eed762204729ad0